requirements.in

```diff
@@ -0,0 +1,40 @@
+# main dependencies
```

Comment:
Curious, what is the difference between these dependencies and those listed in `pyproject.toml`?

Reply:
This `requirements.in` is just the "source" of our constraints file.

And the dependencies in pyproject.toml are the final package dependencies (when installed from pip for example).

I will open an issue about that, because I'm not happy about a single `requirements.txt` as a constraints file, for both our main and our extra dependencies.